src/ProjectService.Models.Dto/Role.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.ProjectService.Models.Dto
+{
+    public class Role
+    {
+        public string Name { get; set; }
+        public string Description { get; set; }
+    }
```

Comment:
Why is it for your task?

Reply:
So, we don`t need model of database?

Reply:
Not use in any command 
![image](https://user-images.githubusercontent.com/38541098/97858422-2c9b7580-1d10-11eb-90ef-0fddc674a6c1.png)

Reply:
Where it use, besides mapper?